content/english/articles/effective-gnops/onboarding-testnet-validator/index.md

```diff
@@ -1,11 +1,85 @@
 level: Intermediate
-author: None
-summary: "This give a set of information on onboarding a validator node: how to provision the right server and connect it to the testnet"
+author: d4ryl00
+summary: "This give a set of information on onboarding a validator node: how to register as a validator operator (valoper) in the registry and how to submit a validator proposal"
 ---
 
-## TBD
+# Setting Up a Validator Node in gno.land
+
+## Overview
+
+Validators on **gno.land** are expected to demonstrate their technical expertise and alignment with the project by making continuous and meaningful contributions. **gno.land** abstracts validator management into the `r/sys/vals` realm as a form of a smart contract for modularity.
+
+This guide walks you through the process of registering your validator node into the validator set with a smart-contract. It assumes that you already have an operational validator node running on the testnet. If not, follow [this guide](https://gnops.io/articles/guides/remote-chain/).
+
+## Step 1: Creating a `Valoper` object
+
+To be added to the validator set, you must create a **Valoper** object and register it in the `r/gnoland/valopers` realm using the `Register` function. This object allows you to demonstrate to **GovDAO members** why you should be accepted as a validator.
+
+### Information to Include in the `Valoper` object
+
+Be sure to follow the instructions inside the valopers realm. This is an example of the details you should provide in your description:
+
+1. **Validator Name** – Your unique identifier.
+2. **Networks You Are Currently Validating** – Include your total Assets Under Management (AuM).
+3. **Links to Your Digital Presence** – Website, social media, etc.
+4. **Contact Details** – How others can reach you.
+5. **Why You Are Interested in Validating on gno.land** – Your motivation and goals.
+6. **Contributions to gno.land** – Past contributions or plans for future contributions.
+
+_Note:_ You can update your `Valoper` object later using the `Update` helper functions such as `UpdateDescription`.
+
+## Step 2: Registering Your Validator
+
+Once your `Valoper` object is prepared, register it using `gnokey` with the following command:
+
+```sh
+gnokey maketx call \
+    -pkgpath "gno.land/r/gnoland/valopers" \
+    -func "Register" \
+    -gas-fee 1000000ugnot \
+    -gas-wanted 30000000 \
+    -send "20000000ugnot" \
+    -broadcast \
+    -chainid "dev" \
+    -args "<moniker>" \
+    -args "<description>" \
+    -args "<validator_address>" \
+    -args "<public_key_validator>" \
+    -remote "https://rpc.gno.land:443" \
+    <key-name>
+```
+
+Replace `<moniker>`, `<description>`, `<validator_address>`, `<public_key_validator>`, and `<key-name>` with your actual values.
+
+## Step 3: Submitting the Proposal
+
+Once your `Valoper` object is ready, you, as the validator, can create a proposal to add yourself to the validator set. Use the following command:
```

Comment:
`anyone can submit a proposal to add you to the validator set`

Reply:
done in https://github.com/gnoverse/gnops/commit/6f4e1f77b426dc6f69c79fd9afd974e064d944f4